panels/export.py

```diff
@@ -0,0 +1,69 @@
+from bpy.props import *
+from bpy.types import (Panel, PropertyGroup)
+
+from ..operators import (ExportUploadOperator, OAuthOperator)
+from ..globals import APP_CATEGORY
+
+SOURCE_TYPES = [
+    ("3D_MODEL", "3D Model", "Models designed through blender", "FILE_3D", 1),
+    ("3D_CAPTURE", "3D Capture",
+     "Model created through photogrammetry processes", "CAMERA_DATA", 2)
```

Comment:
> Model created through photogrammetry processes

Just realized the doc in the REST API is pretty bad.  Let's change this to something like:

Model created through 3D capture such as scanning, LIDAR, photogrammetry, etc...